### ISSUE TITLE
conceal color cannot be changed 

### ISSUE THREAD

**Issue by xarthurx** [arcticicestudio/nord-vim#211]
I'd like use the settings:
```
hi Conceal ctermfg=NONE
hi Conceal ctermbg=NONE
hi Conceal guifg=NONE
hi Conceal guibg=NONE
```

to make concealed chars and words similar to the background.
And it seems there's a bug for this fantastic nord theme:

**Nord:**
![image](https://user-images.githubusercontent.com/1921878/86403807-b8dc5400-bcae-11ea-8a61-0f9df7614993.png)

**OceanicNext:**
![image](https://user-images.githubusercontent.com/1921878/86403827-c2fe5280-bcae-11ea-9908-0b89928d2558.png)

There's an old issue posted here, but seems not resolved:
https://github.com/arcticicestudio/nord-vim/issues/149

And I found some post here:
kaicataldo/material.vim#12
for the 'material` theme, as they add some defined colour to the scheme.

Could we also do similar things for Nord?

**Comment by pithuene** [comment]
I was facing the same problem today but found a solution in https://github.com/vim-pandoc/vim-pandoc-syntax/issues/258. 
Adding the following to my config resolved the weird highlighting.

```
au Colorscheme * hi! link Conceal Number
```

This is on macOS with Neovim in iTerm2 and the conceal is coming from vim-pandoc-syntax.
Hope it helps.

**Comment by xarthurx** [comment]
@pithuene  This is brilliant!
Thanks and it works.

**Comment by tpoisot** [comment]
Just a heads up, this is _not_ working for me on neovim / termite.

**Comment by arcticicestudio** [comment]
@xarthurx @pithuene @tpoisot Thanks for contributions :+1:
The problem has been documented in #256 ⇄ #261 (⊶ f3f28b93) and will be shipped in [version `0.16.0`][1].

Thanks for your patience, the large amount of tasks for the Nord project are really time consuming and free time is kind of rare.

[1]: https://github.com/arcticicestudio/nord-vim/projects/22
